engines/buried/avi_frames.cpp

```diff
@@ -0,0 +1,199 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * Additional copyright for this file:
+ * Copyright (C) 1995 Presto Studios, Inc.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#include "buried/avi_frames.h"
+#include "buried/buried.h"
+#include "buried/graphics.h"
+
+#include "common/system.h"
+#include "graphics/surface.h"
+#include "video/avi_decoder.h"
+
+namespace Buried {
+
+AVIFrames::AVIFrames(const Common::String &fileName, uint cachedFrames) {
+	_maxCachedFrames = 0;
+	_video = 0;
```

Comment:
I think it would be more readable is null pointers were set to nullptr instead of 0